tests/it/c/can_find_partial_paths_in_file.rs

```diff
@@ -5,8 +5,9 @@
 // Please see the LICENSE-APACHE or LICENSE-MIT files in this distribution for license details.
 // ------------------------------------------------------------------------------------------------
 
-use std::collections::HashSet;
+use std::collections::BTreeSet;
```

Comment:
The switch to `BTreeSet` is needed so that the output is ordered, which is needed to have something that can be diffed by `pretty_assertions`.